tests/fsharp/core/printing/z.output.test.1000.stderr.bsl

```diff
@@ -1,6 +1,6 @@
 
   #blaaaaaa // blaaaaaa is not a known command;;
-  ^
+  ^^^^^^^^^
```

Comment:
Why did this change, not that it's not a good change it is.

Reply:
Yes bug fix in pars.fsy I noticed while doing the `#netfx` thing